src/reference/asciidoc/amqp.adoc

```diff
@@ -2854,7 +2906,7 @@ Defaults to a `RabbitAdmin` that will declare all non-conditional elements.
 | missingQueuesFatal
 (missing-queues-fatal)
 
-| Starting with _version 1.3.5_, `SimpleMessageListenerContainer` has this new property.
+a| Starting with _version 1.3.5_, `SimpleMessageListenerContainer` has this new property.
```

Comment:
See [cell formatting](http://asciidoctor.org/docs/user-manual/#cell).

It is needed so the code block within this cell is rendered properly - compare the 1.5.2 release with this snapshot.